api/app/responses/api.py

```diff
@@ -215,15 +227,26 @@ def send_confirmation(self, user, response):
             else:
                 event_description = event.get_description('en')
 
-            emailer.email_user(
-                'confirmation-response-call' if event.event_type == EventType.CALL else 'confirmation-response',
+            if admin_user:
```

Comment:
Since we have access to the `AppUser` object (as the `user` variable here) and we know the `event` we're dealing with let's actually use both those together and then we can remove the `admin_user=None` parameter on this `send_confirmation` method.

```python
if user.is_event_admin(event.id):
```